NET10/Demo.Web.PerformanceTests/Scripts/run-ratelimit-benchmarks.sh

```
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# Rate Limiting 성능 및 부하 테스트 실행 스크립트
+
+echo "=== Rate Limiting Performance and Load Testing ==="
+echo "Starting Rate Limiting benchmarks..."
+
+# 프로젝트 디렉토리로 이동
+cd "$(dirname "$0")/.."
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)